index.html

```diff
@@ -9,37 +9,160 @@
-            <article>               
-                <h1>I'm Charles Nice to meet you!</h1>
-                <p>
-                    My Name is Okoye Charles and I’m a software developer!
-                     I can help you build a product ,
-                    feature or website Look through some of my work and experience! 
-                    If you like what you see and need help on a project, I am available
-                     for hire.
-                </p>
-            </article>
-        
-            <div class="social-header">
-                <p>LET'S CONNECT</p>
-            </div>
-            <nav class="social-links">
-                <ul>
-                    <li><a href="https://twitter.com/OkoyeCharles_"><img src="images/Vector.png" alt="Twitter Icon"></a></li>
+      <div class="home-wrap">
+        <div class="first-section">
+          <header>
+            <nav>
+                <a href="index.html" class="logo">Charles</a>
+                <img src="images/Union.png" alt="MenuBar">
+            </nav>
+          </header>
+          <section class="content">
+            <h1>I'm Charles Nice to meet you!</h1>
+              <article>               
+                <h2 id="invisible">.</h2>
+                  <p>
+                      My Name is Okoye Charles and I’m a software developer!
+                       I can help you build a product ,
+                      feature or website Look through some of my work and experience! 
+                      If you like what you see and need help on a project, I am available
+                       for hire.
+                  </p>
+              </article>
+          
+              <div class="social-header">
+                  <p>LET'S CONNECT</p>
+              </div>
+              <nav class="social-links">
+                  <ul>
+                    <li><a href="https://twitter.com/OkoyeCharles_"><img src="images/Github.png" alt="Twitter Icon"></a></li>
                     <li><a href="https://www.linkedin.com/in/charles-okoye-633374236/"><img src="images/Linkedinicon.png" alt="LinkedIn"></a></li>
                     <li><a href="mailto:okoyecharles509@gmail.com"><img src="images/Gmail.png" alt="Gmail"></a></li>
-                    <li><a href="https://github.com/OkoyeCharles"><img src="images/Github.png" alt="Github"></a></li>
+                    <li><a href="https://github.com/OkoyeCharles"><img src="images/Vector.png" alt="Github"></a></li>
                     <li><a href="#"><img src="images/Fingers.png" alt="Fingers"></a></li>
+                  </ul>
+              </nav>
+          </section>
+        </div>
+      </div>
+        <div class="work-section">
+          <div class="first-card">
+            <div class="card-image"><img src="images/FirstCover.png" alt="FirstCover-Image"></div>
+            <div class="card-content">
+              <article>
+                <h1>Tonic</h1>
+                <div class="card-time">
```

Comment:
-[ ] [REQUIRED] The all project title (e.g., "Tonic") use a header tag `hx`. Please make changes!

Reply:
this not clear what tag exactly

Reply:
<h1>